ccm_web/client/src/models/FeatureUIData.tsx

```diff
@@ -22,11 +30,18 @@ const mergeSectionCardProps: FeatureUIProps = {
   route: '/merge-sections'
 }
 
-const gradebookToolsCardProps: FeatureUIProps = {
-  data: gradebookToolsProps,
+const canvasGradebookCardProps: FeatureUIProps = {
+  data: canvasGradebookToolsProps,
   icon: <LibraryBooksOutlinedIcon fontSize='large' />,
   component: MergeSections,
-  route: '/gradebook'
+  route: '/gradebook-canvas'
+}
+
+const thirdPartyGradebookCardProps: FeatureUIProps = {
+  data: thirdPartygradebookToolsProps,
+  icon: <PostAddOutlinedIcon fontSize='large' />,
+  component: MergeSections,
+  route: '/gradebook-other'
```

Comment:
```suggestion
  route: '/gradebook-thirdparty'
```

Reply:
sorry didn't save that file

Reply:
though its could just as well be named /aasdfwaefqwefasdf